imagetest/cmd/wrapper/main.go

```diff
@@ -24,6 +24,9 @@ func main() {
 	}
 	log.Printf("FINISHED-BOOTING")
 	defer func() {
+		if willRebootDuringTest, err := utils.GetMetadataAttribute("willRebootDuringTest"); err != nil {
```

Comment:
You are not actually using willRebootDuringTest other than printing it, I would leave it out of this PR and only introduce it in the followup PR where we actually use it.

Reply:
done